### DISCUSSION TITLE
Disable animation or change it ?

### DISCUSSION THREAD

**Discussion by iangregsondev** [kurkle/chartjs-chart-matrix#71] [Q&A] [answered]
Hi,

Great chart, i was looking at this one https://chartjs-chart-matrix.pages.dev/samples/time.html

The animation kind of grows from the top left to the bottom right.

Is there any way to disable it or change the animation type ?

I was looking the code/variables in the above link but I didn't see anything that could be changed.

Thanks in advance

**Comment by kurkle** [comment, marked as answer]
The animations are configured as in any chart.js chart: https://www.chartjs.org/docs/latest/configuration/animations.html

TL;DR
`animations: false` in options

You can also try something like `animations: {x: {duration: 0}},`, to disable only animations on x-direction (= the boxes drop down)

**Comment by iangregsondev** [comment]
Thanks. I will take a look
